pkg/backend/display/object_diff.go

```diff
@@ -806,7 +805,7 @@ func (p *propertyPrinter) printPrimitivePropertyValue(v resource.PropertyValue)
 			p.printPropertyValue(vv)
 			return
 		}
-		p.write("%q", v.StringValue())
```

Comment:
Because users might depend on scripting `pulumi stack output` CLI commands, would this change affect the behavior of `pulumi stack output --json` or `pulumi stack output kubeconfig` in a program like the following?

```yaml
name: whatever
runtime: yaml
outputs:
  kubeconfig: |
    contexts:
    - context:
        cluster: docker-desktop
        user: docker-desktop
      name: docker-desktop
    current-context: docker-desktop
    kind: Config
    preferences: {}
    users: []
    # other stuff here
```

I'd expect in both cases, I should be able to get the full YAML document.

Reply:
Yes, this should just be for summary views. Tests to check that would be good

Reply:
`pulumi stack output` should still show the full output, but let's definitely add tests

Reply:
@Frassle realized this change only applies to `pulumi up` when the object diff is computed, so added a flag `show-full-output` in case a user wants it on `pulumi up`, but it does not touch `pulumi stack output`